server/src/modules/maps/maps.controller.ts

```diff
@@ -46,7 +47,7 @@ export class MapsController {
         return this.mapsService.Get(mapID);
     }
 
-    @Post()
+    @Post('')
```

Comment:
What's the need for this, doesn't Nest treat these two identically?

Reply:
Yes it does. I don't remember why I did that, does it need changing?

Reply:
Doesn't matter, changed now anyway